src/pauli_frames.jl

```diff
@@ -120,6 +121,24 @@ function apply!(frame::PauliFrame, op::sMRZ) # TODO sMRY, and faster sMRX
     return frame
 end
 
+function apply!(frame::PauliFrame, op::PauliMeasurement)
+    # this is inspired by ECC.naive_ancillary_paulimeasurement. Not sure if it's better to import and call that.
```

Comment:
```suggestion
    # this is inspired by ECC.naive_syndrome_circuit
```

Better to keep it the way you did it here, as this is non-allocating while `naive_syndrome_circuit` would create a temporary array